test/unit/test_account.py

```diff
@@ -405,6 +405,40 @@ def test_default_env_channel(self):
                 service = FakeRuntimeService()
                 self.assertEqual(service.channel, channel)
 
+    def test_save_private_endpoint(self):
+        """Test private endpoint parameter."""
+
+        # Test without URL
+        AccountManager.save(
+            filename=_TEST_FILENAME,
+            name=_DEFAULT_ACCOUNT_NAME_IBM_CLOUD,
+            token=_TEST_IBM_CLOUD_ACCOUNT.token,
+            instance=_TEST_IBM_CLOUD_ACCOUNT.instance,
+            channel="ibm_cloud",
+            overwrite=True,
+            set_as_default=True,
+            private_endpoint=True,
+        )
+
+        account = AccountManager.get(filename=_TEST_FILENAME)
+        self.assertTrue(account.private_endpoint)
+
+        # If URL is passed in, private_endpoint is ignored
+        AccountManager.save(
+            filename=_TEST_FILENAME,
+            name=_DEFAULT_ACCOUNT_NAME_IBM_CLOUD,
+            token=_TEST_IBM_CLOUD_ACCOUNT.token,
+            instance=_TEST_IBM_CLOUD_ACCOUNT.instance,
+            url=_TEST_IBM_CLOUD_ACCOUNT.url,
+            channel="ibm_cloud",
+            overwrite=True,
+            set_as_default=True,
+            private_endpoint=True,
+        )
+
+        account = AccountManager.get(filename=_TEST_FILENAME)
+        self.assertFalse(account.private_endpoint)
```

Comment:
If we are changing the `url` with the location, the `private_endpoint` param must be taken into account, so you can remove this test.